src/charm.py

```diff
@@ -163,63 +134,85 @@ def _on_install(self, _: InstallEvent) -> None:
         self.unit.status = MaintenanceStatus("Building srsLTE")
         shell(SRS_ENB_UE_BUILD_COMMAND)
 
-        self.unit.status = MaintenanceStatus("Generating configuration files")
+        self.unit.status = MaintenanceStatus("Copying example configuration files")
         copy_files(origin=CONFIG_ORIGIN_PATHS, destination=CONFIG_PATHS)
 
-        self.unit.status = MaintenanceStatus("Generating systemd files")
+        self.unit.status = MaintenanceStatus("Configuring srs env service")
         self._configure_srsenb_service()
-        self._configure_srsue_service()
 
         service_enable(SRS_ENB_SERVICE)
-        self._stored.installed = True
+        self._set_peer_data("installed", True)
```

Comment:
Shouldn't we check if the peer relation is joined? Is it guaranteed to be there when `on_install`?

Reply:
Yeah I don't think we should replace all the "stored states" with peer relation data. Ideally, we have no peer relation data and no stored state and we just "check the thing that needs to be checked" without relying on stored value.

Reply:
I do agree with removing all the "states" possible if we can "check the thing that needs to be checked". However, the `mmm_addr` needs to be stored somewhere, and the peer relation seems like a good place, doesn't it?

Reply:
Removed `peer_data` here